### DISCUSSION TITLE
Mix vs. Style

### DISCUSSION THREAD

**Discussion by leoafarias** [btwld/mix#64] [Ideas]
As we move forward with stabilizing the API would like to open the discussion about changing the main `Mix` API to be named `Style`. Mix is the ability to combine all these different visual properties and seems the industry has adopted "style" as the naming standard. Also Mix takes a lot of inspiration from styled-theme.

## Current implementation

```dart
final boxMix = Mix(
      height(100),
      rounded(10),
    );
    return Box(
      mix: boxMix,
      child: const TextMix('Custom Widget'),
    );
```

## Proposal

```dart
final boxStyle = Style(
      height(100),
      rounded(10),
    );
    return Box(
      style: boxStyle,
      child: const TextMix('Custom Widget'),
    );
```

This is purely a semantic difference but I believe it might help people feel more at home with Mix.

**Comment by bdlukaa** [comment]
I believe `Mix` is very unique and I like it as it currently is. `Mix` also gives the idea of _mixing_ things, which is possible on the current API.

`Style` is a very common name and it removes the uniqueness of the name `Mix`. Also, it gives the idea of only styling, whereas there can also be transformers, such as scaling, aligning, rotating, blurring, and so on...
